build/build.py

```diff
@@ -969,9 +969,10 @@ def configureCompilerOptions(self):
         # specifications properly.
         # Skipping warning 4512 about being unable to generate an assignment
         # operator (since we often do this intentionally).
+	# warning C4100: '...' : unreferenced formal parameter
         # For warnings, use /W4 because /Wall
         # gives us tons of warnings in the VS headers themselves
-        warningFlags = '/W4 /wd4290 /wd4512'
+        warningFlags = '/W4 /wd4290 /wd4512 /wd4100'
```

Comment:
I've always used the "unreferenced parameter" warning as a not-so-subtle hint saying "maybe you should GET RID of this parameter...after all, you're not using it."  Obviously, that doesn't work for a situation like overriding a virtual function.

My personal preference would be to make code changes to remove the warning; but that would seem like a (very) low priority item.  And in the meantime, this warning (and others) might hide something more important.